R/twidlr-gamlss.R

```diff
@@ -0,0 +1,54 @@
+
+
+
+#' data.frame-first formula-second method for \code{\link[gamlss]{gamlss}}
+#'
+#' This function passes a data.frame, formula, and additional arguments to
+#' \code{\link[gamlss]{gamlss}}.
+#'
```

Comment:
As a third paragraph (leaving line 8 empty and starting line 9), mention that this function will save a copy of the data frame used by the model in order to safely predict new data. You can expand to mention the root of this problem and memory issues if you wish. So long as the user is aware of this happening and why.